internal/pkg/cli/env_delete.go

```diff
@@ -348,6 +372,80 @@ func (o *deleteEnvOpts) ensureRolesAreRetained() error {
 	return nil
 }
 
+// emptyBuckets returns nil if buckets were deleted successfully. Otherwise, returns the error.
+func (o *deleteEnvOpts) emptyBuckets() error {
+	envStack, err := o.rg.GetResources(&resourcegroupstaggingapi.GetResourcesInput{
```

Comment:
Why don't we use `(d *EnvDescriber) Describe()` in the `describe` pkg?

Reply:
Because this allows us to find resources not in the stack, so that we can warn the user that they have a hanging resource.

Reply:
Now using `EnvDescriber` for the env stack